src/govuk/components/character-count/character-count.yaml

```diff
@@ -59,6 +59,10 @@ params:
   type: object
   required: false
   description: HTML attributes (for example data attributes) to add to the textarea.
+- name: countMessageClasses
```

Comment:
Should this be

`countMessage` or `message` with a nested `classes` option?

Could allow for updates without breaking changes in the future.


```javascript
{{ govukCharacterCount({
  label: {
    text: "Can you provide more detail?"
  },
  countMessageClasses: 'app-extra-class'
}) }}
```
vs

```javascript
{{ govukCharacterCount({
  label: {
    text: "Can you provide more detail?"
  },
  countMessage: {
    classes: 'app-extra-class'
  }
}) }}
```

Reply:
I think I'd tend towards the nested object, as it's likely we'll need to make the text customisable in the future (somehow) so that it can be overridden if you're using a different language.